### DISCUSSION TITLE
Logging issue?

### DISCUSSION THREAD

**Discussion by modezz** [wiggal/GEN24_Ladesteuerung#51] [Q&A] [answered]
Hallo Wiggal,

warum ist hier z.B. um 08:05 oder 08:50 so ein harter cut? Sqlite hat die Daten von allen 5 bzw in der nacht von allen 11 minuten?
![2301](https://github.com/wiggal/GEN24_Ladesteuerung/assets/38989365/50289fe3-e2e4-4363-a442-a57627e58b59)
![2401](https://github.com/wiggal/GEN24_Ladesteuerung/assets/38989365/48f27ebc-8466-4e73-91b6-4eec9e7a5303)

**Comment by wiggal** [comment]
sieht so aus als würde zwischen 9:00 und 13:50 kein Logging geschrieben.
Poste mal die Cronjobs

**Comment by modezz** [comment]
nope die cronjobs sind ok, anbei der ausschnitt der sqlite welcher durchgängig ist....
![sqlite](https://github.com/wiggal/GEN24_Ladesteuerung/assets/38989365/abacedca-3b46-42d9-a2ac-b8d7d5a46d57)

**Reply by wiggal** [reply]
Die Zeitpunkte sind so unregelmäßig, als würden zwei Cronjobs loggen. Da der WR die Zähler nur alle 5 Minuten schreibt, werden nur Daten dargestellt, die einen Abstand von mindestens 4Minuten haben

**Reply by modezz** [reply]
Jo es läuft der 5 min job vom controller - schreiben und ein 11 min job 24/7 fürs logging - problem?

**Reply by wiggal** [reply]
ja, auch schreiben logged.

**Reply by modezz** [reply]
ok ich nehme den 11 min job mal raus un gggf paase ich die wr schreibgrenzen an!

Danke!

**Reply by wiggal** [reply, marked as answer]
oder den Loggin cronjob in der Zeit in der der schreiben job nicht läuft eintragen

**Reply by wiggal** [reply]
Hallo modezz,

ich habe die Auswahl des 5 Minuten Intervalls verändert (ich hoffe verbessert), nun müssten deine Daten vom 23.01. dargestellt werden.

du musst die Datei 7_funktion_Diagram.php austauschen

**Comment by modezz** [comment]
Wie immer TOP - danke!!
24.01.2014 im vergleich
![2401_new](https://github.com/wiggal/GEN24_Ladesteuerung/assets/38989365/34714281-95d8-4762-8772-8aabf18918fb)

Alt:
![2401](https://github.com/wiggal/GEN24_Ladesteuerung/assets/38989365/2852e031-13e6-4613-9824-c1e585521683)
